testing/trino-tests/src/test/java/io/trino/memory/TestMemoryManager.java

```diff
@@ -115,6 +116,12 @@ public void testOutOfMemoryKiller()
                 .buildOrThrow();
 
         try (DistributedQueryRunner queryRunner = createQueryRunner(TINY_SESSION, properties)) {
+            queryRunner.installPlugin(new BlackHolePlugin());
```

Comment:
`This may be because OOM killer has 5s minimal delay`
This will be dropped (as does not play well with task-level retries. (part of https://github.com/trinodb/trino/pull/11800).

Reply:
Will the test be updated to use smaller delay? This could make the test run shorter.

Still, the change here should remain. If query can succeed in low number of seconds, this feels racy.